transport/dgit/client.go

```diff
@@ -50,44 +49,131 @@ func Default() (*Client, error) {
 func NewClient(ctx context.Context, basePath string) (*Client, error) {
 	var err error
 	c := &Client{ctx: ctx}
-	dir := path.Join(basePath, protocol)
-	c.tupelo, c.nodestore, err = clientbuilder.Build(ctx, dir)
+	dir := path.Join(basePath, constants.Protocol)
+	c.Tupelo, c.Nodestore, err = clientbuilder.Build(ctx, dir)
```

Comment:
@cap10morgan just to note related to this, I added `Client.DagStore()` method in https://github.com/quorumcontrol/tupelo-go-sdk/pull/212/files#diff-7f6280b5f587f41a183254e1b182f3c9R100 so we don't have to keep passing around both vars everywhere